app/src/main/java/com/infomaniak/drive/utils/AccountUtils.kt

```diff
@@ -193,16 +195,28 @@ object AccountUtils : CredentialManager {
         }
     }
 
-    suspend fun removeUser(context: Context, userRemoved: User) {
-        userDatabase.userDao().delete(userRemoved)
-        FileController.deleteUserDriveFiles(userRemoved.id)
+    suspend fun removeUserAndDeleteToken(context: Context, user: User) {
+        initInfomaniakLogin(context).deleteToken(
+            KDriveHttpClient.getHttpClient(user.id),
+            user.apiToken,
+            onError = {
+                Log.e("deleteTokenError", "Api response error : ${LoginActivity.getLoginErrorDescription(context, it)}")
+            },
+        )
```

Comment:
So if the request takes too long to answer, the person will be blocked in the app?
It has to be in another thread.

Reply:
done